assets/src/apps/authoring/components/PropertyEditor/schemas/lesson.ts

```diff
@@ -52,21 +59,63 @@ export const lessonUiSchema = {
     'ui:ObjectFieldTemplate': CustomFieldTemplate,
     'ui:title': 'Screen Size',
     width: {
-      classNames: 'col-6'
+      classNames: 'col-6',
     },
     height: {
-      classNames: 'col-6'
-    }
+      classNames: 'col-6',
+    },
   },
   Appearance: {
     'ui:ObjectFieldTemplate': CustomFieldTemplate,
-    'ui:title': 'Lesson Appearance'
+    'ui:title': 'Lesson Appearance',
   },
   ScoreOverview: {
     'ui:ObjectFieldTemplate': CustomFieldTemplate,
-    'ui:title': 'Score Overview'
-  }
+    'ui:title': 'Score Overview',
+  },
 };
 
+// we don't have the actual theme urls yet,
+// they will likely come from somehwere else
+const themeMap: { [key: string]: string } = {
+  'url to new theme': 'Light Responsive',
+};
+
+export const transformModelToSchema = (model: any) => {
```

Comment:
this has a specific type
```suggestion
export const transformModelToSchema = (model: PageState) => {
```

Reply:
I am not sure it's this one, I think it's actually more of LessonModel which doesn't yet exist